src/commands/ProtectionsCommands.ts

```diff
@@ -33,6 +33,38 @@ export async function execEnableProtection(roomId: string, event: any, mjolnir:
     }
 }
 
+export async function execSetProtection(roomId: string, event: any, mjolnir: Mjolnir, parts: string[]) {
+    const [protectionName, ...settingParts] = parts[2].split(".");
+    const protection = PROTECTIONS[protectionName];
+    if (protection !== undefined) {
```

Comment:
Nit: Matter of style, but I'm personally used to `if (protection === undefined) return;`